doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,11 +8,47 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
 
-The scaling configuration specifies distributed training properties like the number of workers or the
-resources per worker.
+Key concepts
+------------
+To understand how to configure scale best, this sections reviews concepts that relate
+to Ray clusters and distributed training.
+
+.. figure:: ../images/train_cluster_overview.png
+    :align: center
+    :width: 400px
+
+- **Cluster**: This is the :ref:`Ray cluster <cluster-index>` on which the training is
+  executed. There is only one Ray cluster your code interacts with.
+- **Node**: A node is one machine connected to the Ray cluster.
+  For instance, this can be a Physical Machine, a Virtual Machine, or a Kubernetes Pod.
+  The :ref:`Head Node <cluster-head-node>` is usually where you
+  execute your script (but you can also execute it on any other node).
+  The cluster can comprise of one to many hundreds of nodes. Each node has specific
+  :ref:`Resources <core-resources>`, such as CPUs, GPUs, memory, or
+  :ref:`custom resources <custom-resources>`.
+- **Driver**: This is the script you are executing. It's called the driver as it is
+  responsible for scheduling the Ray actors and tasks (it's "driving" the execution).
+- **Trainer**: The trainer object you instantiate in Ray Train. This lives in two
+  places: First, you create it in the Driver script. Then, when you call
+  :meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`, a copy will be
+  created as a :ref:`Ray Actor <actor-key-concept>`.
+- **(Train) Worker**: In the context of Ray Train, we often refer to training workers.
+  Technically, these are also :ref:`Ray Actors <actor-key-concept>` that execute
+  your training loop. You can specify an arbitrary number of workers. The number of workers
+  can be more or less than the number of nodes. In the figure, we show two
+  training workers per node.
+
+Unfortunately, the term "worker" is used differently in different contexts. In Ray
+Train, a "Worker" will always be a "Training worker" (an Actor executing your training
+loop), unless we explicitly refer to a "Worker node".
+
+Configuring scale in Ray Train (``ScalingConfig``)
+--------------------------------------------------
```

Comment:
Looking at the side nav it's not clear to me how the information in this page should be structured. 

<img width="290" alt="image" src="https://github.com/ray-project/ray/assets/3967392/b615b56a-4d31-4e16-be79-2e1543cf041c">

Right now it feels like there could potentially be a lot of overlap between different sections. In particular "Configuring scale in Ray Train ( ScalingConfig )" and "How many nodes, workers, and resources should I use?" both feel like a superset of the individual sections.

Is there a different way to organize this (maybe hierarchical, combining sections, renaming sections, etc.) so that it is more logical where the user should find information, and where developers should add new information?

Reply:
I've updated the headlines a bit. Let's maybe schedule a quick call to go over this